tests/js/t16822.nim

```diff
@@ -0,0 +1,13 @@
+# bug #16822
+var scores: seq[(set[char], int)] = @{{'/'} : 10}
+
+var x: set[char]
+for item in items(scores):
+  x = item[0]
+
+doAssert x == {'/'}
+
+for (chars, value) in items(scores):
+  x = chars
+
+doAssert x == {'/'}
```

Comment:
Should use a different `x` here, the loop could be a nop

Reply:
was done